servers/src/main/java/tachyon/master/lineage/meta/LineageFileState.java

```diff
@@ -15,9 +15,56 @@
 
 package tachyon.master.lineage.meta;
 
+import tachyon.exception.ExceptionMessage;
+
 /**
  * The state of a lineage file.
  */
 public enum LineageFileState {
-  CREATED, COMPLETED, PERSISTED, PERSISENCE_REQUESTED, LOST
+  CREATED, COMPLETED, PERSISTED, PERSISENCE_REQUESTED, LOST;
+
+  /**
+   * @return the corresponding protobuf {@link tachyon.proto.journal.Lineage.LineageFileState}
+   */
+  public tachyon.proto.journal.Lineage.LineageFileState toProtobuf() {
```

Comment:
for InodeDirectory we call this pair of methods `fromJournalEntry` and `toJournalEntry` ... it would be good to be consistent